draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1348,14 +1348,15 @@ Section Header
             hashing algorithm, while the following octets contain the actual
             hash, whose size depends on the hashing algorithm, and hence from
             the value in the first octet. The hashing algorithm can be: 2s
-            complement (algorithm octet = 0, size=XXX), XOR (algorithm octet =
+            complement (algorithm octet = 0, size = XXX), XOR (algorithm octet =
             1, size=XXX), CRC32 (algorithm octet = 2, size = 4), MD-5
-            (algorithm octet = 3, size=XXX), SHA-1 (algorithm octet = 4,
-            size=XXX). The hash covers only the packet, not the header added
-            by the capture driver: this gives the possibility to calculate it
-            inside the network card. The hash allows easier comparison/merging
-            of different capture files, and reliable data transfer between the
-            data acquisition system and the capture library.</t>
+            (algorithm octet = 3, size = 16), SHA-1 (algorithm octet = 4,
+            size = 20), Toeplitz (algorith octet = 5, size = 4). The hash covers
```

Comment:
Typo - "algorithM octet".

Should each algorithm type have a link to a definition of the algorithm, if available?

Reply:
Thanks, darn typos... now I need to find out how to fix the problem... I assume I need to close the pull request and create a new one, but I'm not sure if that will also kill this comment thread. 

Links to the definition is something we could add, yes. The common ones like MD5 and SHA1 probably would not really require it, but Toeplitz is indeed a bit "under the radar". I can prepare something and create another pull request.

Reply:
> now I need to find out how to fix the problem... I assume I need to close the pull request and create a new one

You *shouldn't* have to.  [About Pull Requests](https://help.github.com/en/articles/about-pull-requests) says:

> Once you've created a pull request, you can push commits from your topic branch to add them to your existing pull request. These commits will appear in chronological order within your pull request and the changes will be visible in the "Files changed" tab.

So, as far as I know, "make another commit to fix the typo and do `git push`" should work, and "make a `git commit -a --amend` to fix the typo and do `git push`" might also work.

I've only rarely done git pulls (the main git repositories on which I work are libpcap, tcpdump, and the pcapng spec, and I'm a core committer to all of them, so I just directly `git push` from my machine), so I don't know if the second of those would work (the equivalent - `git commit -a --amend` followed by `git review` - works on the Wireshark repository).

Reply:
Interesting... you're right, the git push for the typo from my local PC automatically made it appear here, apparently. Thx!

Reply:
if you use git commit -a --amend, need to use git push -f(orce) because rewrite history... (but also work..)